renderer/HardwareOpenGL.cpp

```diff
@@ -430,24 +422,39 @@ int opengl_Setup(oeApplication *app, const int *width, const int *height) {
         LOG_WARNING << "No parameter for -display given";
       } else {
         int arg_index = atoi(arg_index_str);
-        int display_count = SDL_GetNumVideoDisplays();
+        int display_count = 0;
+        SDL_GetDisplays(&display_count);
```

Comment:
We might have a little mem leak here. [SDL_GetDisplays](https://wiki.libsdl.org/SDL3/SDL_GetDisplays) states that the array returned "should be freed with [SDL_free](https://wiki.libsdl.org/SDL3/SDL_free)() when it is no longer needed".